frontend/src/metabase/css/dashboard.css

```diff
@@ -331,8 +331,13 @@
   display: none;
 }
 
+/* disable dc default behavior */
 .dc-chart rect.bar:hover {
-  fill-opacity: 1;
+  fill-opacity: 1 !important;
```

Comment:
This gives me the icks. Is there any way around using `!important` or is the DC styling too aggressive?